OneSignalSDK/unittest/src/test/java/com/test/onesignal/MainOneSignalClassRunner.java

```diff
@@ -3202,7 +3233,7 @@ public void onOSSubscriptionChanged(OSSubscriptionStateChanges stateChanges) {
 
    @Test
    public void shouldAddPermissionObserverIfConsentNotGranted() throws Exception {
-      OneSignal.setRequiresUserPrivacyConsent(true);
```

Comment:
Maybe we should still pass in a `boolean` param so it's easier it read? I saw this always sets it `true`

Reply:
Actually we default to false, we have this   
`ShadowOneSignalRestClient.setRemoteParamsRequirePrivacyConsent(); `

for setting the privacy consent to true

Reply:
I know, thats what im saying, it seems confusing to call a method that toggles privacy consent without actually passing in a boolean param. Defaulting to false is fine but if we want to be clear in our UnitTests what state its in we should have to do something like this

``` Java
ShadowOneSignalRestClient.setRemoteParamsRequirePrivacyConsent(true);
ShadowOneSignalRestClient.setRemoteParamsRequirePrivacyConsent(false);
```

Just calling `ShadowOneSignalRestClient.setRemoteParamsRequirePrivacyConsent` is kind of confusing

Reply:
Im okay if we keep this how it is, but I still think its confusing if we always set true with this method. What if we want to toggle back to false in a specific UnitTest?